cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -779,6 +779,43 @@ TEST_F(TestProjector, TestModZero) {
   EXPECT_ARROW_ARRAY_EQUALS(exp_mod, outputs.at(0));
 }
 
+TEST_F(TestProjector, TestFindInSet) {
+  // schema for input fields
+  auto field0 = field("f0", arrow::utf8());
+  auto field1 = field("f1", arrow::utf8());
+  auto schema = arrow::schema({field0, field1});
+
+  // output fields
+  auto field_find = field("find_in_set", arrow::int32());
+
+  // Build expression
+  auto find_expr =
+      TreeExprBuilder::MakeExpression("find_in_set", {field0, field1}, field_find);
+
+  std::shared_ptr<Projector> projector;
+  auto status = Projector::Make(schema, {find_expr}, TestConfiguration(), &projector);
+  EXPECT_TRUE(status.ok()) << status.message();
+
+  // Create a row-batch with some sample data
+  int num_records = 4;
+  auto array0 = MakeArrowArrayUtf8({"A", "", "Z", "A"}, {true, true, true, true});
```

Comment:
Make a test passing some of the values as null, just to ensure that the response is null too.